client-app/pages/compare-products/compare-products.vue

```diff
@@ -0,0 +1,211 @@
+                >
+                  <i class="fas fa-times text-red-500"></i>
+                </div>
+                <VcImage
+                  :src="product.imgSrc"
+                  :alt="product.name"
+                  class="w-full h-full object-cover object-center"
+                  lazy
+                />
+              </div>
+            </router-link>
+
+            <!-- Product title -->
+            <router-link
+              :to="`/${SeoUrl.Product}/${product.id}`"
+              class="text-[color:var(--color-link)] font-extrabold text-sm mb-3 flex-grow line-clamp-3 overflow-hidden cursor-pointer"
+            >
+              {{ product.name }}
+            </router-link>
+
+            <!-- Product price -->
+            <div class="flex flex-col md:flex-row items-baseline text-sm mb-3">
+              <div class="w-1/2 font-bold text-xs" v-if="!isMobile">
+                {{ $t("pages.compare.main_block.price_label") }}
+              </div>
+              <div class="md:w-1/2">
+                <span class="text-green-700 font-extrabold"><VcPriceDisplay :value="product.price?.actual" /></span>
+                {{ $t("common.suffixes.per_item") }}
+              </div>
+            </div>
+
+            <AddToCart :product="product" />
+          </div>
+        </div>
+
+        <!-- Properties block -->
+        <div
+          class="items-start md:items-center space-x-5 px-5 lg:px-0 border-b border-gray-100 lg:border-0"
+          v-for="(values, key, index) in computedProperties"
+          :key="index"
+          :class="!isMobile ? 'even:bg-gray-50 flex' : productsIds.length >= 3 ? 'inline-flex' : 'flex'"
+        >
+          <div class="w-1/6 pl-8 font-extrabold text-sm" v-if="!isMobile">{{ key }}</div>
+          <div class="w-32 flex-shrink-0 lg:flex-shrink md:w-48 py-5" v-for="(value, index) in values" :key="index">
+            <span v-if="isMobile" class="block font-extrabold text-sm">{{ key }}</span>
+            {{ value.value }}
+          </div>
+        </div>
+        <div class="flex items-center space-x-5 px-5 lg:px-0">
+          <div class="w-1/6" v-if="!isMobile"></div>
+          <div v-for="product in products" :key="product.id" class="w-32 flex-shrink-0 lg:flex-shrink md:w-48">
+            <!-- Product price -->
+            <div class="flex flex-col md:flex-row items-baseline text-sm my-4">
+              <div class="w-1/2 font-bold text-xs" v-if="!isMobile">
+                {{ $t("pages.compare.main_block.price_label") }}
+              </div>
+              <div class="md:w-1/2">
+                <span class="text-green-700 font-extrabold"><VcPriceDisplay :value="product.price?.actual" /></span>
+                {{ $t("common.suffixes.per_item") }}
+              </div>
+            </div>
+            <AddToCart :product="product" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script setup lang="ts">
```

Comment:
Divide template and script by 1 empty line

Reply:
fixed